newDB.sql

```diff
@@ -1,97 +1,110 @@
-create table Branch (
-  IFS_Code varchar(255) not null primary key,
-  Branch_Name varchar(255) not null,
-  City varchar(255) not null,
-  Branch_Address varchar(255) not null,
-  State varchar(255) not null,
-  Country varchar(255) not null,
-  PIN_Code varchar(255) not null
+create table Branches (
+  id bigserial primary key,
+  ifs_code varchar(255) not null unique,
+  name varchar(255) not null,
+  country varchar(255) not null,
+  state varchar(255) not null , -- check constraint for state and city
+  city varchar(255) not null,
+  address varchar(255) not null,
+  postal_code int not null,
+  created_at timestamp without time zone not null default now(),
+  updated_at timestamp without time zone
+  check(updated_at > created_at)
 );
 
-create table Account_Master (
+create table Customers (
   id bigserial primary key,
-  Account_Type varchar(255) not null,
-  prefix varchar(255) not null,
-  Min_balance decimal(10,2) not null,
-  interest decimal(10,2) not null
+  branch_id bigint not null references Branches(id),
+  first_name varchar(255) not null,
+  last_name varchar(255) not null,
+  country text not null,
+  state text not null, -- check constraint for state and city
+  city text not null,
+  address text [] not null,
+  postal_code int not null,
+  created_at timestamp without time zone not null default now(),
+  updated_at timestamp without time zone,
+  check(updated_at > created_at)
 );
 
-create table Customer (
-  CustomerID int not null primary key,
-  IFS_Code varchar(255) not null references Branch(IFS_Code),
-  F_Name varchar(255) not null,
-  L_Name varchar(255) not null,
-  Login_ID varchar(255) not null,
-  Acc_Pass varchar(255) not null,
-  Trans_Pass varchar(255) not null,
-  Acc_Status varchar(255) not null,
-  City varchar(255) not null,
-  State varchar(255) not null,
-  Country varchar(255) not null,
-  Acc_Open_Date date not null,
-  Last_login date not null
-);
+create index branch_idx on Customers(branch_id);
```

Comment:
is the index name in accordance with naming contention from out DB session?

Reply:
by mistake I created this index with the wrong naming convention other all indexes are made with the correct naming convention.